transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -114,12 +122,40 @@ export class SmallWebRTCTransport extends Transport {
     super();
     this._iceServers = opts.iceServers ?? [];
     this._waitForICEGathering = opts.waitForICEGathering ?? false;
-    this._webrtcUrl = opts.webrtcUrl ?? opts.connectionUrl ?? null;
     this.audioCodec = opts.audioCodec ?? null;
     this.videoCodec = opts.videoCodec ?? null;
 
-    if (opts.connectionUrl) {
-      logger.warn("connectionUrl is deprecated, use webrtcUrl instead");
+    const _webrtcUrl = opts.webrtcUrl ?? opts.connectionUrl ?? null;
+    if (_webrtcUrl) {
+      if (opts.connectionUrl) {
+        logger.warn(
+          `connectionUrl is deprecated, use webrtcRequestParams instead: { webrtcRequestParams: { endpoint: ${opts.connectionUrl} } }`,
+        );
+      } else {
+        logger.warn(
+          `webrtcUrl is deprecated, use webrtcRequestParams instead: { webrtcRequestParams: { endpoint: ${opts.webrtcUrl} } }`,
+        );
+      }
+      if (opts.webrtcRequestParams) {
+        logger.warn(
+          "Both webrtcUrl/connectionUrl and webrtcRequestParams provided. Using webrtcRequestParams.",
+        );
+      } else {
+        if (typeof _webrtcUrl === "string") {
+          this._webrtcRequest = { endpoint: _webrtcUrl };
+        } else {
+          logger.warn(`webrtcUrl is not a string. Ignoring: ${_webrtcUrl}`);
+        }
+      }
+    }
+    if (opts.webrtcRequestParams) {
```

Comment:
The only case where this wouldn’t be true is if we haven’t received either webrtcRequestParams, webrtcUrl, or connectionUrl, right ? 

But I guess that’s okay, since we can receive it directly in the connect method, right ?